### PR TITLE
about_us file created

### PR DESCRIPTION
**Pull request by Naeembaloch21**

### CODE REVIEW

File: about_us.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<?php
+include('layouts/header.php')
+?>
+<?php
+include('layouts/navbar.php')
+?>
+
```
Comment (Farazbhai) [inline about_us.php]: khali line nhi choro

File: about_us.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+            <p class="m-0">Home / About Us 01</p>
+        </div>
+    </div>
+</div>
+<div class="d-flex justify-content-center mt-5">
+</div>
+<h1 class="text-center mt-3">Welcom to Auteur</h1>
+<h3 class="text-center mt-3">A MONTHLY BOOKS REVIEW PUBLICATION</h3>
+<div class="container mt-5">
+    <p>Imagine two men facing each other, pointing past one another. One is pointing at a tornado that is coming,
```
Comment (Farazbhai) [inline about_us.php:22]: format this code
hahaha

File: about_us.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        <h3 class="ms-5">We point past each other with our words, arguing as though we are looking at the same facts
+            and experiences.</h3>
+    </div>
+    <p class="mt-5">We want to prove our words are the right ones, instead of learning to look at what the other’s
+        words are pointing at. Words are seductive, and for all their undeniable usefulness, they also can lead us
+        away from understanding when we focus on
+        them, when we make them more important than the truth they are meant to point at.</p>
+</div>
+<div class="container">
+    <video src="https://www.youtube.com/watch?v=eH2WNtL5ong "></video>
```
Comment (Farazbhai) [inline about_us.php]: remove this video

File: about_us.php [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    <div class="container-fluid mb-5 ">
+        <div class="row ">
+            <div class="col-sm-4 mt-4">
+                <img src="assets/images/single-image-16.jpg " width="100% " alt="Error:404">
```
Comment (Farazbhai) [inline about_us.php]: remove Error:404 and add image not found

File: about_us.php [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+            </div>
+            <div class="col-sm-4 mt-4 ">
+                <img src="assets/images/single-image-18.jpg " width="100% " alt="Error:404">
```
Comment (Farazbhai) [inline about_us.php]: remove Error:404 and add image not found

File: about_us.php [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        </div>
+    </div>
+</div>
+<div class="container-fluid " style="background-color: rgba(211, 211, 211, 0.253); ">
+    <h3 class="text-danger text-center ">TESTIMONIALS</h3>
+    <h1 class="text-center ">Read Reviews by My Readers</h1>
+    <div class="container ">
+        <div class="row ">
+            <div class="col-sm-6 mt-5 ">
+                <img class="rounded-circle " src="assets/images/testimonial-01-140x140.jpg " alt="Error:404">
```
Comment (Farazbhai) [inline about_us.php]: remove Error:404 and add image not found

File: about_us.php [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+                <p class="fw-bold ">" Auteur is a monthly book review publication distributed to 400,000 avid
+                    readers through subscribing bookstores & public libraries. "</p>
+                <p class="fw-bold ">Vladimir Nabokov</p>
+                <p>/Reporter</p>
+            </div>
+            <div class="col-sm-6 mt-5 ">
+                <img class="rounded-circle " src="assets/images/testimonial-02-140x140.jpg " alt="Error:404">
```
Comment (Farazbhai) [inline about_us.php]: remove Error:404 and add image not found

File: about_us.php (hunk 2 above)
Comment (Akram-baloch) [reply about_us.php:22]: hahaha

Comment (MuslimJaan) [reply about_us.php:22]: hahaha

File: about_us.php (hunk 3 above)
Comment (Akram-baloch) [reply about_us.php]: agree

File: about_us.php (hunk 5 above)
Comment (Akram-baloch) [reply about_us.php]: agree

File: about_us.php (hunk 6 above)
Comment (Akram-baloch) [reply about_us.php]: follow this instruction

**Review by Naeembaloch21 — COMMENTED** [review]
requested changes.

**Pull request merged**